### PR TITLE
docs: add icon story to progress bar

### PR DESCRIPTION
**Pull request by JoeyDjr**
Add the story Icon of the `ProgressBar` component.

(It is better to merge the PR https://github.com/okp4/ui/pull/404 before that one.)

### CODE REVIEW

File: stories/atoms/components/progressBar/ProgressBar.stories.mdx [hunk 1]
```diff
@@ -204,3 +205,45 @@ By default, the current value is not formatted and the progress value is display
...
+
+## Icon
+
+The `icon` property allows you to display an icon (or any `JSX.Element` at the right of the progress bar).<br />
+Icons can be provided to give more visual information such as a hourglass during the progress time or a cross if the progress is interupted or a check when the progress is completed.
+
+<Canvas>
+  <Story name="Icon">
+    {() => {
+      return (
```
Comment (fredericvilcot) [inline stories/atoms/components/progressBar/ProgressBar.stories.mdx:217]: As it's not used, you don't need this lambda function anymore 😉

File: stories/atoms/components/progressBar/ProgressBar.stories.mdx [hunk 2]
```diff
@@ -204,3 +205,45 @@ By default, the current value is not formatted and the progress value is display
...
+            icon={<Icon name="clock" />}
+          />
+          <ProgressBar
+            label="An error occured"
+            minValue={0}
+            maxValue={15}
+            icon={<Icon name="cross" />}
+          />
+          <ProgressBar
+            label="Lesson completed !"
```
Comment (fredericvilcot) [inline stories/atoms/components/progressBar/ProgressBar.stories.mdx:239]:
```suggestion
            label="Lesson completed!"
```

File: stories/atoms/components/progressBar/ProgressBar.stories.mdx [hunk 3]
```diff
@@ -204,3 +205,45 @@ By default, the current value is not formatted and the progress value is display
     }}
   </Story>
 </Canvas>
+
+## Icon
+
+The `icon` property allows you to display an icon (or any `JSX.Element` at the right of the progress bar).<br />
```
Comment (fredericvilcot) [inline stories/atoms/components/progressBar/ProgressBar.stories.mdx:211]:
```suggestion
The `icon` property allows you to display an icon (or any `JSX.Element`) at the right of the progress bar.<br />
```

**Review by fredericvilcot — CHANGES_REQUESTED** [review]

File: stories/atoms/components/progressBar/ProgressBar.stories.mdx [hunk 4]
```diff
@@ -204,3 +205,42 @@ By default, the current value is not formatted and the progress value is display
     }}
   </Story>
 </Canvas>
+
+## Icon
+
+The `icon` property allows you to display an icon (or any `JSX.Element`) at the right of the progress bar.<br />
```
Comment (fredericvilcot) [inline stories/atoms/components/progressBar/ProgressBar.stories.mdx:211]: @JoeyDjr there is a surviving `<br />` tag here 😜

**Review by fredericvilcot — APPROVED** [review]

**Review by ccamel — APPROVED** [review]

**Pull request merged**